src/store/modules/errors.js

```diff
@@ -0,0 +1,17 @@
+const defaulState = {
+  message: '',
+  showError: false,
+}
+
+export default {
+  namespaced: true,
+  state: defaulState,
+  mutations: {
+    setMessage(state, message) {
+      state.message = message
+    },
+    setShowError(state, show) {
+      state.showError = show
+    },
+  },
+}
```

Comment:
It is necessary to create a subscribe error class, similar to the following (include it in another file):

```suggestion
}

class ConnectionError extends Error {
  constructor(message) {
    super(message);
    this.name = 'ConnectionError'
  }
}
```